extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -380,18 +374,17 @@ export class AmpLightboxGallery extends AMP.BaseElement {
    * @private
    */
   buildDescriptionBox_() {
-    this.descriptionBox_ = this.win.document.createElement('div');
-    this.descriptionBox_.classList.add('i-amphtml-lbg-desc-box');
-    this.descriptionBox_.classList.add('i-amphtml-lbg-standard');
+    this.descriptionBox_ = htmlFor(this.doc_)`
+      <div class="i-amphtml-lbg-desc-box i-amphtml-lbg-standard">
+        <div class="i-amphtml-lbg-desc-text"></div>
+        <div class="i-amphtml-lbg-desc-mask"></div>
+      </div>`;
 
-    this.descriptionTextArea_ = this.win.document.createElement('div');
-    this.descriptionTextArea_.classList.add('i-amphtml-lbg-desc-text');
+    this.descriptionTextArea_ = childElement(this.descriptionBox_,
```

Comment:
how about `scopedQuerySelector(this.descriptionBox_, 'i-amphtml-lbg-desc-text');`?

Reply:
Shorter is good. +1